dev-server/ui-playground/Patterns.js

```diff
@@ -0,0 +1,60 @@
+import { toChildArray } from 'preact';
```

Comment:
Again, this is to support the pattern library, but isn't central to this PR if you're not interested.